src/telemetry.ts

```diff
@@ -0,0 +1,79 @@
+import * as vscode from "vscode";
+import axios from "axios";
+import {v4 as uuidv4} from 'uuid';
+import * as util from "./util";
+
+const LAST_UPDATED_DATE = "lastUpdatedDate";
+const INSTALLATION_DATE = "installDate";
+const INSTALLATION_UUID = "installUUID";
+const URL = "https://stats.grafana.org/vscode-usage-report";
+
+/*
+ * Sends a single anonymous telemetry call once per month, allowing tracking of
```

Comment:
Says once per month here, but the test is `differentDay` so it really is once a day, right?

Reply:
well spotted, fixed.